spec.bs

```diff
@@ -567,7 +567,95 @@ For now, see [https://github.com/wicg/observable#operators](https://github.com/w
+                 unsubscribe from |notifier| since we got all we need from it, and no longer need it
+                 to keep producing values. We also unsubscribe from |sourceObservable|, because it
+                 no longer needs to produce values that get plumbed through this method's returned
+                 |observable|, because we're manually ending the subscription to |observable|, since
+                 |notifier| finally produced a value.
+
+              1. |sourceObservable| either {{Subscriber/error()}}s or {{Subscriber/complete()}}s
+                 itself. In this case, we unsubscribe from |notifier| since we no longer need to
+                 listen for values it emits in order to determine when |observable| can stop
+                 mirroring values from |sourceObservable| (since |sourceObservable| ran to
+                 completion by itself). Unsubscribing from |sourceObservable| isn't necessary, since
+                 its subscription has exhausted itself.
+          </div>
+
+       1. Let |signal| be the result of [=creating a dependent abort signal=] from the list
+          «|controller|'s [=AbortController/signal=], |subscriber|'s [=Subscriber/signal=]», using
+          {{AbortSignal}}, and the [=current realm=].
+
+       1. Let |notifierObserver| be a new [=internal observer=], initialized as follows:
+
+          : [=internal observer/next steps=]
+          :: 1. Run |subscriber|'s {{Subscriber/complete()}} method.
+
+             1. [=AbortController/Signal abort=] |controller|.
+
+          : [=internal observer/error steps=]
+          :: 1. Run |subscriber|'s {{Subscriber/complete()}} method.
+
+             1. [=AbortController/Signal abort=] |controller|.
+
+          Note: We do not specify [=internal observer/complete steps=], because if the |notifier|
+          {{Observable}} completes itself, we do not need to complete the |subscriber| associated
+          with the |observable| returned from this method. Rather, the |observable| will continue to
+          mirror |sourceObservable| uninterrupted.
+
+       1. Let |options| be a new {{SubscribeOptions}} whose {{SubscribeOptions/signal}} is |signal|.
+
+       1. <a for=Observable lt="subscribe to an Observable">Subscribe</a> to |notifier| given
+          |notifierObserver| and |options|.
+
+       1. If |subscriber|'s [=Subscriber/active=] is false, then return.
+
+          Note: This means that |sourceObservable|'s [=Observable/subscribe callback=] will not even
+          get invoked once, if |notifier| synchronously emits a value. If |notifier| synchronously
+          "completes" though (without emitting a "next" or "error" value) then we proceed to
+          subscribe to |sourceObservable|, and |observable| will mirror it uninterrupted. In that
+          case, |notifier| has lost the opportunity to ever stop the returned |observable| from
+          mirroring |sourceObservable|.
```

Comment:
This bit took a couple of read throughs to understand. I wonder if it can be made more explicit or clear?

Reply:
I removed the last sentence since it wasn't adding much value, and I tried to clear up some of the earlier text a little bit.